disnake/permissions.py

```diff
@@ -970,6 +972,14 @@ def use_soundboard(self) -> int:
         """
         return 1 << 42
 
+    @flag_value
+    def use_external_sounds(self) -> int:
```

Comment:
this should be added to `Permissions.voice` as well.

Reply:
[`11a1626` (#997)](https://github.com/DisnakeDev/disnake/pull/997/commits/11a1626fb9592d207a53d4573e5c9d1d470b6de4)